logging/cloud-client/src/test/java/com/example/logging/QuickstartSampleIT.java

```diff
@@ -0,0 +1,69 @@
+/*
+  Copyright 2016, Google, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+package com.example.logging;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.cloud.logging.Logging;
+import com.google.cloud.logging.LoggingOptions;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+/**
+ * Tests for quickstart sample.
+ */
+@RunWith(JUnit4.class)
+@SuppressWarnings("checkstyle:abbreviationaswordinname")
+public class QuickstartSampleIT {
+
+  private ByteArrayOutputStream bout;
+  private PrintStream out;
+
+  private static final void deleteMyLog() {
+    Logging logging = LoggingOptions.getDefaultInstance().getService();
+
+    logging.deleteLog("my-log");
+  }
+
+  @Before
+  public void setUp() {
+    deleteMyLog();
+
+    bout = new ByteArrayOutputStream();
+    out = new PrintStream(bout);
+    System.setOut(out);
+  }
+
+  @After
+  public void tearDown() {
+    System.setOut(null);
+    deleteMyLog();
+  }
+
+  @Test
+  public void testQuickstart() throws Exception {
+    QuickstartSample.main("my-log");
```

Comment:
The test is reading stdout, not Cloud Logging. While it'd be nice if the quickstart also showed how to read logs, I think that's beyond the scope of a quickstart. @jmdobry thoughts?